piIOComm.c

```diff
@@ -429,8 +429,7 @@ void piIoComm_writeSniff1A(EGpioValue eVal_p, EGpioMode eMode_p)
 
 void piIoComm_writeSniff1B(EGpioValue eVal_p, EGpioMode eMode_p)
 {
-	if (piDev_g.machine_type == REVPI_CORE ||
-	    piDev_g.machine_type == REVPI_CORE_SE) {
+	if (piDev_g.pibridge_supported && !piDev_g.only_left_pibridge) {
```

Comment:
I don't think we ever come here in the !piDev_g.pibridge_supported case, so checking only for !piDev_g.only_left_pibridge is sufficient.

Reply:
Same for the three subsequent changes.